test/unit/calculators/energy_grants_calculator_rates_query_test.rb

```diff
@@ -0,0 +1,17 @@
+require_relative "../../test_helper"
+
+module SmartAnswer::Calculators
+  class EnergyGrantsCalculatorRatesQueryTest < ActiveSupport::TestCase
+    setup do
+      @rates ||= RatesQuery.from_file('energy_grants_calculator').rates
+    end
+
+    context 'for 2016/17' do
+      should 'be 5th May 1953 for winter fuel payments threshold' do
+        Timecop.travel('30 Sep 2016')
```

Comment:
I'd prefer we didn't use `Timecop.travel` here. I think we should be passing the date to `RatesQuery#rates` instead. See [register_a_birth_rates_query_test.rb](https://github.com/alphagov/smart-answers/blob/master/test/unit/calculators/register_a_birth_rates_query_test.rb) for an example of how we're doing this elsewhere.